nautobot/extras/views.py

```diff
@@ -1828,7 +1828,7 @@ def get_extra_context(self, request, instance):
                 "device_type",
             )
             ipaddress = instance.ip_addresses.select_related(
-                "vrf",
+                # "vrf",
```

Comment:
> Notes:
> 
> * I think there was unanimous agreement that there shouldn't be a VRF property on IPAddress so I've updated places that might have caused issues in testing in UI (instead of commenting out), there might be some utility in a helper method to get the parent prefix's vrf

Oh yeah, for sure, I stopped at the places that were causing utter breakage in other views. I didn't actually complete the necessary work for IPAddress.